### ISSUE TITLE
Offer alternative to Google ReCapctha (or shortcodes)

### ISSUE THREAD

**Issue by MAthez** [elementor/elementor#8079]
## Prerequisites
- [x ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I would like to avoid using Google ReCaptcha, because it is causing prohibitive RGPD-related issues. When implementing RGPD in cookie alerts, one has to offer users to avoid third party/ marketing cookies, and many of them do. As a result, ReCAPTCHA is not working, and users fail to submit forms. 

**Describe the solution you'd like**
I'd like to be able use alternatives in Elementor Pro Form. 

**Describe alternatives you've considered**
- I tried alternative reCAPTCHA, such as WPBruiser, Invisible ReCaptcha ou Really Simple Captcha. Theyuy offer either extensions for forms (but not Elementor), or shortcodes. But Elementor Pro Form HTML item does not seem to support shortcodes. 
- Alternatively, something like CF7 Quiz item could do. But Elementor Pro Form only offers a "Number" item that will be easily passed by bots.

**Comment by blaue-mango** [comment]
Any updates on this? Or does anyone know a plugin that could integrate this feature? 
GDPR doesn't really allow us to use reCaptcha.
The spam mails are really annoying. (Eric Jones seems to have much time on his hands.) 

This is an unbearable situation.

The really bad thing is we can't offer a solution to our customers since there simply is none. 

- Opting in via consent is not a good solution. 
- Using alternatives such as CF7 might be an option, but kinda destroys the purpose of the page builder.

**Comment by MAthez** [comment]
> Any updates on this? 
> The really bad thing is we can't offer a solution to our clients since there simply is none.

Never got an update on this. But I found a workaround that enabled me to stick to elementor form: using a simple question as an anti-spam filter, based on the "number" type. 
It kind works well. See here: https://www.mathezfreight.com/en/contact/

**Comment by blaue-mango** [comment]
> using a simple question as an anti-spam filter, based on the "number" type. It kind works well. See here: https://www.mathezfreight.com/en/contact/

Good idea. I tried this now. Unfortunately, it doesn't stop Eric Jones from submitting the form. 
I tried a simple question "What is 4 + 3" with the only number to be allowed to be 7. Just like you did.
But Jones simply ignores it and fills in a "0" and is able to submit. So the HTML5 input check doesn't work for him.

![image](https://user-images.githubusercontent.com/32264292/105166338-a6751b00-5b17-11eb-81d4-604296d6d0ea.png)

**Comment by MAthez** [comment]
You have to
- use a question that is not easy to guess. Rather than arithmetic operations, I used "how many months in January", or "how many days in a year", which people have no trouble to answer but bots may not. 
- you can, and must, use the settings in the number type of field, using "min" and "max" set to your expected value
![image](https://user-images.githubusercontent.com/19684250/105178988-09bb7900-5b29-11eb-980a-f5d9a47983ee.png)

**Comment by ribtoks** [comment]
You can take a look at [Private Captcha](https://github.com/PrivateCaptcha/PrivateCaptcha) - it's main point is being GDPR-ready out of the box. Does not collect any user data, you can also self-host it. It also has integrations with different languages.
